ui/nvui/src/chess.ts

```diff
@@ -567,15 +567,13 @@ export function possibleMovesHandler(color: Color, fen: () => string, pieces: ()
     const $boardLive = $('.boardstatus');
     const $pieces = pieces();
     const myTurnFen = color === 'white' ? 'w' : 'b';
-    const opponentTurnFen = color === 'white' ? 'b' : 'w';
 
     const $btn = $(ev.target as HTMLElement);
     const $pos = (($btn.attr('file') ?? '') + $btn.attr('rank')) as SquareName;
 
     // possible ineffecient to reparse fen; but seems to work when it is AND when it is not the users' turn.
-    const possibleMoves = chessgroundDests(
-      Chess.fromSetup(parseFen(fen().replace(' ' + opponentTurnFen + ' ', ' ' + myTurnFen + ' ')).unwrap()).unwrap()
-    )
+    const $fen = fen() + ' ' + myTurnFen;
```

Comment:
This variable name isn't ideal. `$var` usually signifies Cash objects. Why not just leave it inlined?

Reply:
Oops. Yeah, good point. I'll fix that.